Tasks/CondaAuthenticateV0/task.json

```diff
@@ -0,0 +1,39 @@
+{
+ "id": "543c39b2-11e9-4de8-8b9b-94eea1339891",
+ "name": "CondaAuthenticate",
+ "friendlyName": "Conda authenticate (for task runners)",
+ "description": "Authentication task for the conda client",
+ "helpUrl": "",
+ "helpMarkDown": "",
+ "category": "Package",
+ "author": "Microsoft Corporation",
+ "version": {
+     "Major": 0,
+     "Minor": 220,
+     "Patch": 0
+ },
+ "runsOn": [
+    "Agent",
+    "DeploymentGroup"
+ ],
+ "minimumAgentVersion": "2.144.0",
+ "instanceNameFormat": "conda Authenticate $(configFile)",
+ "inputs": [
+     {
+         "name": "configFile",
+         "type": "filePath",
+         "label": ".condarc file to authenticate",
+         "required": true,
+         "helpMarkDown": "Path to the .condarc file with the PAT for the channels you want to authenticate"
+     }
+ ],
+ "execution": {
+    "Node10": {
```

Comment:
Add node16 equivalent and verify that it works too

Reply:
Done!